### ISSUE TITLE
Failed to mount docker volume

### ISSUE THREAD

**Issue by alfredsgenkins** [scandipwa/create-magento-app#9]
The following error was shown during recent installation on my Linux Mint:

```
      ↓ Pulling container images [SKIPPED]
    ✖ Starting containers
  ◼ Setup magento
  ◼ Starting php-fpm
  ◼ Open browser
ERROR! docker: Error response from daemon: failed to mount local volume: mount /home/scandiweb/Projects/create-scandipwa-app/runtime-packages/cma/node_modules/.create-scandipwa-app-cache/nginx/conf.d:/var/snap/docker/common/var-lib-docker/volumes/cma_nginx-data/_data, flags: 0x1000: no such file or directory.

```

**Comment by ejnshtein** [comment]
Please run `docker volume inspect cma_nginx-data` and show output here.
Seems like an issue with nginx data volume.

**Comment by ejnshtein** [comment]
Turns out that volume mount point does not exist anymore, but volume is present and new volume has a similar name to the previous one. And it dies.

I see a few options "how to fix this":
1. Manually delete old volumes. or use **cleanup** command.
2. Introduce project **prefixes** so all docker containers, volumes and networks will have 100% unique name. They will be stored in some config file inside the system, perhaps global one will be a safer approach. (using [configstore](https://www.npmjs.com/package/configstore))
2.1. Switch from **exec** library to spawn docker commands to use [docker-modem](https://www.npmjs.com/package/docker-modem) to communicate with docker directly.

**Comment by ejnshtein** [comment]
Fixes in v1.1.6
